src/backend/egl/mod.rs

```diff
@@ -1,30 +1,30 @@
 //! Common traits and types for egl rendering
 //!
-//! Large parts of this module are taken from
-//! [glutin src/api/egl](https://github.com/tomaka/glutin/tree/044e651edf67a2029eecc650dd42546af1501414/src/api/egl/)
+//! This module has multiple responsibilities related to functionality provided by libEGL:
+//! - Initializing EGL objects to
+//!   - initialize usage of EGL based [`WlBuffer`](wayland_server::protocol::wl_buffer::WlBuffer)s via `wl_drm`.
+//!   - initialize OpenGL contexts from.
+//!   - Import/Export external resources to/from OpenGL
```

Comment:
It seems there is one too many layers of lists here? The topmost list has only one item.